lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -206,15 +286,17 @@ def stage_input(self, params):
 
         return input_directory
 
-    def parse_genbank(self, file_path, params):
+    def parse_genbank(self, workspace_id, file_path, params, genome_obj):
```

Comment:
Is this called from outside this class? If so, those calls will need updates. If not we should make the method private

Reply:
Ideally only 2 functions (import_genbank and import_genbank_mass) in this module should be public as they are used by impl.py file. All others should be private. 

One thing I am not sure is if ppl would access them through non client call.

Reply:
Only way to tell is by searching the codebase. It's safe to assume that other repos aren't calling methods that aren't defined in the spec in their code

Reply:
All public ones are only used in this module. Indeed, they should be private.

Reply:
👍

Reply:
Is there a reason `validate_params` is public?

Reply:
:+1:

Reply:
Yeah, should be private.
Most of the modules inside `core` have their own validate_params function. 
Not sure why ppl made it public